src/utils/cluster_utils.cc

```diff
@@ -30,15 +30,16 @@ Eigen::VectorXd bayesmix::cluster_estimate(
 
   // Compute mean
   std::cout << "(Computing mean dissimilarity... " << std::flush;
-  Eigen::MatrixXd mean_diss = bayesmix::posterior_similarity(alloc_chain);
+  Eigen::MatrixXd mean_diss =
+      bayesmix::posterior_similarity(alloc_chain.cast<double>());
```

Comment:
VectorXd -> VectorXi seems most natural since the clustering is integer-values, while saving a VectorXd to file writes a vector of doubles. Casting in Eigen is not expansive thankfully